apm-sniffer/apm-sdk-plugin/elastic-job-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticjob/ElasticJobExecutorInterceptor.java

```diff
@@ -36,7 +35,7 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         ShardingContexts shardingContexts = (ShardingContexts) allArguments[0];
         Integer item = (Integer) allArguments[1];
         String operateName = ComponentsDefine.ELASTIC_JOB.getName() + "/" + shardingContexts.getJobName();
-        AbstractSpan span = ContextManager.createEntrySpan(operateName, new ContextCarrier());
+        AbstractSpan span = ContextManager.createLocalSpan(operateName);
```

Comment:
Do you want to analyze? If you want, add `logic span` tag.

Reply:
@tristaZero @terrymanu After this is added, on the SkyWalking UI, you will have the job metrics, besides the traces, including
1. Execution time, including avg, percentile
1. Successful rate, determined through whether the main entrance has an exception
1. What other endpoints(HTTP/gRPC) this task depends on.

This is a thing we called the logic endpoint.

Reply:
It sounds a great feature. Express my thinks to @zhaoyuguang and @wu-sheng .

Reply:
Elastic-job`s had specification logic-span